Taskfile.yml

```diff
@@ -65,4 +70,10 @@ tasks:
       KO_DOCKER_REPO: ghcr.io/stackloklabs/gofetch/server
       VERSION: dev-local
     cmds:
-      - ko build --local --bare ./cmd/server
+      - |
+        if [[ $(uname -m) == "arm64" ]]; then
+          PLATFORM="linux/arm64"
+        else
+          PLATFORM="linux/amd64"
+        fi
+        ko build ./cmd/server --bare --local --platform=$PLATFORM
```

Comment:
Why change this? Ko detects the right platform and does the right thing.

Reply:
Nope, turns out it doesn't :( by default it will run `linux/amd64`. Which was a bit weird because I could have sworn this code worked before.